### PR TITLE
test(client): check console.log

### PR DESCRIPTION
**Pull request by hiroppy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

yes

### Motivation / Use-Case

Just testing `console.log` of the client's code.

### Breaking Changes

no

### Additional Info

Maybe liveReload has a bug... see the snapshot.

### CODE REVIEW

File: test/Client.test.js [hunk 1]
```diff
@@ -229,3 +229,71 @@ describe('Client complex inline script path with sockHost', () => {
...
+        .then(({ page, browser }) => {
+          return new Promise((resolve) => {
+            page.goto('http://localhost:9000/main');
+            page.on('console', ({ _text }) => {
+              res.push(_text);
+            });
+            setTimeout(() => {
+              expect(res).toMatchSnapshot();
+              browser.close().then(resolve);
+            }, 3000);
```
Comment (hiroppy) [inline test/Client.test.js]: Maybe here is flaky tests.

File: test/__snapshots__/Client.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Client console.log hot disabled 1`] = `
+Array [
+  "Hey.",
+  "[WDS] Live Reloading enabled.",
```
Comment (hiroppy) [inline test/__snapshots__/Client.test.js.snap]: Should it be output this line?

File: test/__snapshots__/Client.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+exports[`Client console.log liveReload disabled 1`] = `
+Array [
+  "Hey.",
+]
+`;
+
+exports[`Client console.log liveReload enabled 1`] = `
+Array [
+  "Hey.",
```
Comment (hiroppy) [inline test/__snapshots__/Client.test.js.snap]: We should expect `[WDS] Live Reloading enabled.`

File: test/__snapshots__/Client.test.js.snap (hunk 2 above)
Comment (evilebottnawi) [reply test/__snapshots__/Client.test.js.snap]: Yes

File: test/__snapshots__/Client.test.js.snap (hunk 3 above)
Comment (evilebottnawi) [reply test/__snapshots__/Client.test.js.snap]: Yes, need investigate what is problem

**Comment by hiroppy** [comment]
client-src depends on browsers variables, e.g `window`, `postMessage`. So this code should be tested on browsers. I propose to divide the tests into different types as npm tasks.

**Comment by evilebottnawi** [comment]
@hiroppy jest support jsdom and windows and other tools avaliable, but maybe it is bad idea try to use jsdom because many features are unavailable, so i think it is ok using this approach

**Comment by evilebottnawi** [comment]
Anyway need investigate why some value invalid in tests as i written above

**Comment by hiroppy** [comment]
>  jest support jsdom and windows and other tools avaliable, but maybe it is bad idea try to use jsdom because many features are unavailable, so i think it is ok using this approach

Yeah, I agree with you.

> Anyway need investigate why some value invalid in tests as i written above

I would like to investigate after merging this PR.

**Comment by hiroppy** [comment]
I just fixed lint. PTAL /cc @evilebottnawi

**Comment by hiroppy** [comment]
And I'll start to investigate client code, so I need this pr.

**Pull request merged**